### PR TITLE
Harrison/add scheduled trigger json for test

### PR DESCRIPTION
**Pull request by NgoHarrison**
# Fix prettier formatting and consolidate metadata.json reading logic

## Summary
This PR addresses the nit comment on PR #3033 by consolidating metadata.json reading logic into reusable utility functions, and fixes the failing compile CI check.

**Key changes:**
1. **Fixed prettier formatting** in `project.test.ts` and `triggers.ts` to resolve compile CI failures
2. **Created new utility module** (`ee/vellum_ee/workflows/display/utils/metadata.py`) with consolidated metadata.json reading functions:
   - `get_trigger_edge_id()` - Get stable edge ID for trigger edges
   - `get_entrypoint_edge_id()` - Get stable edge ID for entrypoint edges  
   - `get_regular_edge_id()` - Get stable edge ID for regular node-to-node edges
3. **Refactored `base_workflow_display.py`** to use the new utility functions instead of inline helper functions
4. **Added scheduled trigger support** with proper serialization and test fixtures

The consolidation allows entities to pass in classes and get back IDs, making the metadata.json reading logic reusable and easier to maintain.

## Review & Testing Checklist for Human
- [ ] **Verify edge ID retrieval works correctly** - Test that workflows with metadata.json correctly retrieve stable edge IDs for all edge types (trigger, entrypoint, regular)
- [ ] **Test scheduled trigger serialization** - Verify that scheduled triggers serialize correctly with display data and that the new test fixture passes
- [ ] **Check for regressions** - Ensure existing workflows (especially those with manual and integration triggers) still serialize correctly
- [ ] **Review the metadata utility functions** - Verify the logic for finding workflow roots and loading edge mappings handles edge cases (missing files, malformed JSON, etc.)

### Notes
- The compile CI check now passes ✅
- The test CI check is still failing but was explicitly excluded from scope per user request
- There's a pre-existing mypy type error on line 438 (entrypoint_display assignment) that existed before these changes and is unrelated to this PR
- Session: https://app.devin.ai/sessions/6edf9486dd044995a152d4b33b80599d
- Requested by: harrison@vellum.ai (@NgoHarrison)

### CODE REVIEW

File: ee/vellum_ee/workflows/display/workflows/base_workflow_display.py [hunk 1]
```diff
@@ -190,6 +193,52 @@ def serialize(self) -> JsonObject:
...
+
+        def _load_edges_to_id_mapping(module_path: str) -> Dict[str, str]:
+            try:
+                root = _find_workflow_root_with_metadata(module_path)
+                if not root:
+                    return {}
+                file_path = os.path.join(root.replace(".", os.path.sep), "metadata.json")
+                with virtual_open(file_path) as f:
+                    data = json.load(f)
+                    edges_map = data.get("edges_to_id_mapping")
```
Comment (dvargasfuertes) [inline ee/vellum_ee/workflows/display/workflows/base_workflow_display.py]: [nit] again, would love all of the metadata.json reading logic consolidated (in case we want to move them somewhere else/restructure) so that each of our entities are simply passing in classes and getting back ids

**Review by dvargas92495 — APPROVED** [review]

**Comment by NgoHarrison** [comment]
@codex review

**Comment by NgoHarrison** [comment]
@codex review

File: pyproject.toml [hunk 2]
```diff
@@ -94,7 +94,7 @@ plugins = [
     "vellum.plugins.vellum_mypy",
 ]
 check_untyped_defs = true
-exclude = ["venv", "examples/workflows", "ee/codegen/python_file_merging/tests/fixtures"]
+exclude = ["venv", "examples/workflows", "ee/codegen/python_file_merging/tests/fixtures", "ee/codegen_integration/fixtures/simple_scheduled_trigger"]  #TODO: Uncomment trigger test once we fix mypy issue with graph: https://linear.app/vellum/issue/APO-2127/fix-mypy-issue-of-graph-types
 overrides = [
```
Comment (vincent0426) [inline pyproject.toml]: this will be override in postprocess
